packages/webpack/test/e2e-lockdown-inline.spec.js

```diff
@@ -0,0 +1,29 @@
+const test = require('ava')
+const { scaffold, runScript } = require('./scaffold.js')
+const { makeConfig } = require('./fixtures/main/webpack.config.js')
+
+test.before(async (t) => {
+  const webpackConfig = makeConfig({
+    inlineLockdown: ['app.js'],
+  })
+  await t.notThrowsAsync(async () => {
+    t.context.build = await scaffold(webpackConfig)
+  }, 'Expected the build to succeed')
```

Comment:
the side-effect here is little malodorous to me, but I'm not sure what a better solution would look like.

Reply:
I'm not sure what you mean is a side effect here. scaffold is running the build and returning information it collects. If it fails, we report it as a testsuite failure.